src/storage/mvcc/reader/scanner/mod.rs

```diff
@@ -363,7 +365,11 @@ where
     I: Iterator,
 {
     let seek_key = user_key.clone().append_ts(write_start_ts);
-    default_cursor.near_seek(&seek_key, &mut statistics.data)?;
+    match statistics.load_data_hint() {
+        LoadDataHint::NearSeek => default_cursor.near_seek(&seek_key, &mut statistics.data)?,
+        LoadDataHint::Seek => default_cursor.seek(&seek_key, &mut statistics.data)?,
+    };
+
```

Comment:
The `near_load_data_by_write` function name and comments should be updaterd.

Reply:
Update the comment, but not the function name. I think it's clear enough.